.github/workflows/CI_windows.yml

```diff
@@ -17,6 +17,13 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: configure Pagefile
+        uses: al-cheb/configure-pagefile-action@v1.2
+        with:
+          minimum-size: 8GB
+          maximum-size: 16GB
+          disk-root: "D:"
+
```

Comment:
Please note that this is still under the review process of the security compliance team, as this `configure-pagefile` action is a 3rd party GitHub action from an unverified author.